packages/embeds/embed-core/src/preview.ts

```diff
@@ -1,7 +1,11 @@
-const WEBAPP_URL =
-  import.meta.env.EMBED_PUBLIC_WEBAPP_URL || `https://${import.meta.env.EMBED_PUBLIC_VERCEL_URL}`;
-const EMBED_LIB_URL = import.meta.env.EMBED_PUBLIC_EMBED_LIB_URL || `${WEBAPP_URL}/embed/embed.js`;
-
```

Comment:
Orgs change forced this logic to improve. No need to read from env variables here.